<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KIIT_11535" ignored="0" name="Default test" tests="2" failures="1" timestamp="2021-06-07T13:42:21 IST" time="655.129" errors="0">
  <testcase name="runScenario" time="442.857" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="209.965" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at com.pages.FinalPage.verifyFlightDetails(FinalPage.java:33)
at com.stepdefinitions.MMTStepDefinition.verifyFlightDetails(MMTStepDefinition.java:121)
at âœ½.Verify Booking details(file:///C:/Users/barkha.sinha/Internship_Training/TrainingAssignment/src/test/resources/FeatureFiles/MMTAutomation.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
